examples/sql_bank/main.go

```diff
@@ -49,76 +50,80 @@ func moveMoney(db *sql.DB) {
 			continue
 		}
 		amount := rand.Intn(*maxTransfer)
-		tx, err := db.Begin()
-		if err != nil {
-			log.Fatal(err)
-		}
-		startTime := time.Now()
-		query := fmt.Sprintf("SELECT id, balance FROM accounts WHERE id IN (%d, %d)", from, to)
-		rows, err := tx.Query(query)
-		elapsed := time.Now().Sub(startTime)
-		if elapsed > 10*time.Millisecond {
-			log.Infof("%s took %v", query, elapsed)
-		}
-		if err != nil {
-			if log.V(1) {
-				log.Warning(err)
-			}
-			if err = tx.Rollback(); err != nil {
-				log.Fatal(err)
+
+		switch *transferStyle {
+		case 0:
+			update := `
+UPDATE bank.accounts
+  SET balance = CASE id WHEN $1 THEN balance-$3 WHEN $2 THEN balance+$3 END
+  WHERE id IN ($1, $2)
```

Comment:
This needs a guard against the "from" account going negative. Actually, it looks like the multi-statement version doesn't have this either; it got lost in the transition from `kv_bank`.